release notes/v0.48.0.md

```diff
@@ -158,6 +158,28 @@ const redisClient = new redis.Client({
 ```
 </details>
 
+### Add tracing instrumentation [#3445](https://github.com/grafana/k6/pull/3445)
+
+`k6` now supports a new *traces output* option that allows to configure the output for traces generated during its execution. This option can be set through the `--traces-output` argument in `k6 run` command or by setting the `K6_TRACES_OUTPUT` environment variable.
+
+By default traces output is set to `none`, and currently the only supported output is `otel` which uses the [opentelemetry-go](https://github.com/open-telemetry/opentelemetry-go)'s [Open Telemetry](https://opentelemetry.io/) API and SDK implementations. The format for the `otel` *traces output* configuration is the following:
+
+```
+--traces-output=<endpoint>[,opt1=val1,opt2=val2]
+```
+Where `opt`s can be one of the following options:
+* `proto`: Specifies the protocol to use in the connection to the traces backend. Supports `grpc` *(default)* and `http`.
+* `header.<header_name>`: Specifies an additional header to include in the connection to the traces backend.
+
+Example:
+```
+K6_TRACES_OUTPUT=https://traces.k6.io/v1/traces,proto=http,header.Authorization=Bearer token
+```
+
+This configuration will set up an underlying [*Tracer Provider*](https://github.com/grafana/k6/blob/e603a494f48bc9b106ffbcbdc6f7320c4563946a/lib/vu_state.go#L85-L86) which is exposed to extensions through the [VU interface](https://github.com/grafana/k6/blob/e603a494f48bc9b106ffbcbdc6f7320c4563946a/js/modules/modules.go#L41)'s [*State()*](https://github.com/grafana/k6/blob/e603a494f48bc9b106ffbcbdc6f7320c4563946a/js/modules/modules.go#L57) method. The *Tracer Provider* can be used to initialize [*Tracers*](https://pkg.go.dev/go.opentelemetry.io/otel/trace#Tracer) which are then used for the actual traces generation.
```

Comment:
Maybe all these detail references to the actual code is not necessary here, but I thought it could also be useful for external contributors to k6 or to extensions. Let me know what do you think.

Reply:
Yeah I think this can be dropped as well

Reply:
Yep, we shouldn't replace documentation with release notes. Related to it, do we have documentation?